src/styles.scss

```diff
@@ -1,4 +1,6 @@
+@import '../node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css';
```

Comment:
it must be declared in angular.json

Reply:
its only for I can to see how will be look